app/admin.py

```diff
@@ -225,3 +229,18 @@ def __view_config_to_model_config(self, view_config):
             else:
                 model_config[name] = value
         return model_config
+
+    def __update_repo_status(self, view_config):
+        repo_entity = model.Repo.get_by_key_name(self.env.repo)
+        launch_status = view_config['launch_status']
+        if launch_status == 'staging':
```

Comment:
just by habbit, I'll ask if it's worth putting these hard-coded strings in some constants somewhere?

Reply:
Done.